content/data-protocols-formats/frontend-backend.md

```diff
@@ -1519,6 +1549,16 @@ Then, execute the following SQL statement to disable the protocol debug log afte
 
 ## Summary of Changes since Protocol 3.0
 
+### Protocol 3.15
+- [OAuth 2.0 Authentication](#protocol-315) enhancement: Format change in the [AuthenticationOAuth](#authenticationoauth-r) message to support OAuth browser workflow. 
+- Workload support. Format change in the [StartupRequest](#startuprequest) message. New 'workload' parmeter allowing specification of workload name to be used by workload routing rules.
+- Format change in [WriteFile](#writefile-o) for ODBC driver. New 'protocol_features' parameter called 'extend-copy-reject-info' in [StartupRequest](#startuprequest) message allows ODBC driver users to properly get rejected row information for bulk loaded data through calls to SQLGetDiagRec().
```

Comment:
The second one. ODBC sends 'extend-copy-reject-info' in startup message in order to get the change in the WriteFile format. Technically any other driver could send that parameter, but it would break the parsing of the WriteFile message, and no other driver has a reason for needing the different format right now